### PR TITLE
add default roles to services sa

### PR DESCRIPTION
**Pull request by anakovt**

### CODE REVIEW

File: modules/service-account/vars.tf [hunk 1]
```diff
@@ -10,6 +10,11 @@ variable services {
   description = "Map of IAM Roles to assign to the Services Service Account"
 }
 
+variable common_iam_roles {
+  description = "Default list of IAM Roles to assign to every Services Service Account"
+  type        = list(string)
```
Comment (askoriy) [inline modules/service-account/vars.tf:15]: add default value here - empty list

File: modules/project/main.tf [hunk 2]
```diff
@@ -62,6 +64,7 @@ module "secret_manager_sa" {
   create_service_group   = var.create_secret_manager_group
   service_group_name     = var.service_group_name
   clan_gsuite_group      = var.clan_gsuite_group
+  common_iam_roles       = var.common_iam_roles
```
Comment (askoriy) [inline modules/project/main.tf]: skip this

File: modules/project/main.tf [hunk 3]
```diff
@@ -48,6 +49,7 @@ module "cloudrun_sa" {
   create_service_group   = var.create_cloudrun_group
   service_group_name     = var.service_group_name
   clan_gsuite_group      = var.clan_gsuite_group
+  common_iam_roles       = var.common_iam_roles
```
Comment (askoriy) [inline modules/project/main.tf]: skip this

File: modules/project/main.tf [hunk 4]
```diff
@@ -34,6 +34,7 @@ module "ci_cd_sa" {
   create_service_group   = var.create_ci_cd_group
   service_group_name     = var.service_group_name
   clan_gsuite_group      = var.clan_gsuite_group
+  common_iam_roles       = var.common_iam_roles
```
Comment (askoriy) [inline modules/project/main.tf]: skip this

**Review by Santos92 — APPROVED** [review]
LGTM

File: modules/project/vars.tf [hunk 5]
```diff
@@ -171,6 +171,12 @@ variable services {
   description = "Map of IAM Roles to assign to the Services Service Account"
 }
 
+variable common_iam_roles {
```
Comment (askoriy) [inline modules/project/vars.tf:174]: declare actual roles for services here

File: modules/project/main.tf [hunk 6]
```diff
@@ -76,6 +79,7 @@ module "services_sa" {
   create_service_group   = var.create_services_group
   service_group_name     = var.service_group_name
   clan_gsuite_group      = var.clan_gsuite_group
+  common_iam_roles       = ["roles/monitoring.metricWriter", "roles/logging.logWriter", "roles/monitoring.viewer", "roles/cloudtrace.agent"]
```
Comment (askoriy) [inline modules/project/main.tf]: declare roles in variable and use variable here

**Review by askoriy — COMMENTED** [review]
I'd make more simpler, see comments

**Pull request merged**